background/services/keyring/tests/index.unit.test.ts

```diff
@@ -245,4 +231,53 @@ describe("Keyring Service", () => {
       expect(signed).toBeDefined()
     })
   })
+
+  describe("export secrets", () => {
+    beforeEach(async () => {
+      await keyringService.importSigner({
+        type: SignerTypes.privateKey,
+        privateKey: PK_WALLET_MOCK.privateKey,
+      })
+      await keyringService.importSigner({
+        type: SignerTypes.keyring,
+        mnemonic: HD_WALLET_MOCK.mnemonic,
+        source: "import",
+      })
+    })
+    it("should be able to export private key", async () => {
+      const privateKey = await keyringService.exportPrivateKey(
+        PK_WALLET_MOCK.address
+      )
+
+      expect(privateKey).toBe(PK_WALLET_MOCK.privateKey)
+    })
+    it("should be able to export mnemonic", async () => {
+      const mnemonic = await keyringService.exportMnemonic(
+        HD_WALLET_MOCK.addresses[0]
+      )
+
+      expect(mnemonic).toBe(HD_WALLET_MOCK.mnemonic)
+    })
+    it("should be able to export private key from HD wallet addresses", async () => {
+      const privateKey = await keyringService.exportPrivateKey(
+        HD_WALLET_MOCK.addresses[0]
+      )
+
+      expect(privateKey).toBe(PK_WALLET_MOCK.privateKey) // first address from both mocks is the same
+    })
+    it("should require wallet to be unlocked to export secrets", async () => {
+      keyringService.lock()
+
+      const errorMessage = "KeyringService must be unlocked."
+      const exportMnemonic = async () => {
+        await keyringService.exportMnemonic(HD_WALLET_MOCK.addresses[0])
+      }
+      const exportPrivateKey = async () => {
+        await keyringService.exportPrivateKey(PK_WALLET_MOCK.address)
+      }
+
+      expect(exportMnemonic()).rejects.toThrowError(errorMessage)
+      expect(exportPrivateKey()).rejects.toThrowError(errorMessage)
```

Comment:
Non-blocking comment
There is no difference. However, I have seen that we already use `toThrow`.

```suggestion
      expect(exportMnemonic()).rejects.toThrow(errorMessage)
      expect(exportPrivateKey()).rejects.toThrow(errorMessage)
```